### PR TITLE
September newsletter

### PR DESCRIPTION
**Pull request by wipfli**
Template for the September newsletter. 

@HarelM and @louwers please add a few words about web and native. Thanks!

### CODE REVIEW

**Review by louwers — CHANGES_REQUESTED** [review]
We should not put images in the git repo.

**Comment by nyurik** [comment]
@louwers why not?  If the image is small enough, i don't see any reason to keep it text-only

File: content/news/2023-09-18-maplibre-newsletter-september-2023/index.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+- Support arbitrary base and interval values in RasterDEM sources [#326](https://github.com/maplibre/maplibre-style-spec/issues/326).
+- Change default value for text-font to Noto Sans Regular [#311](https://github.com/maplibre/maplibre-style-spec/issues/311)
+
+Furthermore, there is an ongoing discussion about how to support more writing systems such as Indic scripts with Harfbuzz text shaping. Read more [here](https://github.com/maplibre/maplibre-style-spec/discussions/312).
+
+## MapLibre Native
+
+### Add subtitle...
+
+## MapLibre GL JS
```
Comment (HarelM) [inline content/news/2023-09-18-maplibre-newsletter-september-2023/index.md:42]:
```suggestion
## MapLibre GL JS

We've released version 3.3.1 with a few minor bug fixes related to typings and improved license.txt file handling for packaging tools.

```

**Comment by HarelM** [comment]
I've updated web with minimal text as there wasn't a lot to update...

**Comment by louwers** [comment]
@nyurik Because it will balloon the git repo over time. And git doesn't deal well with binary files. For example, you delete one it will still be part of the git history, and it's a pain to scrum them.

There is also a 1GB GitHub Pages size limit, which we will hit pretty quickly if we make a habit of storing binary blobs on GitHub pages.

There is also a 100 GB bandwidth limit, which sounds like quite a lot, but hitting the front page of Hacker News brings 10k-30k visitors to the site *per day*, so we'd likely hit the limit in that case (although Cloudflare caching may make this a non-issue).

**Review by HarelM — APPROVED** [review]

**Review by louwers — APPROVED** [review]

**Comment by wipfli** [comment]
@tordans did you get a RSS notification for this newsletter now?

**Comment by tordans** [comment]
@wipfli yes, looks great. Thanks for getting this deployed!
Will look into providing the full feed when I find time…

image

**Comment by wipfli** [comment]
Oh nice, well, thanks for setting it up!

**Pull request merged**